DZ_3/zadacha_2.py

```diff
@@ -0,0 +1,19 @@
+# Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+
+# Пример:
+
+# - [2, 3, 4, 5, 6] => [12, 15, 16];
+# - [2, 3, 5, 6] => [12, 15]
+
+
+import random
+n = int(input('Введите размер строки '))
+res = random.sample(range(0, 20), n)
+
+print(res)
+
+prod = []
+for i in range((len(res)+1)//2):
```

Comment:
```suggestion
for i in range(n + 1) // 2):
```
Зачем считать длину, если переменная, по которой создается список определенной длины, у вас уже существует?